docs/common/istio-ambient-mode.md

```diff
@@ -0,0 +1,368 @@
+
+## Introduction to Istio Ambient mode
+
+Ambient mesh was announced in September 2022 and reached a stable state in the Istio 1.24 release. It's core innovation is the separation of Layer 4 (L4) and Layer 7 (L7) processing into two distinct layers. It uses lightweight, shared L4 node proxies and optional L7 proxies, eliminating the need for traditional sidecar proxies in the data plane.
+
+The lightweight, shared L4 node proxy is called ztunnel (zero-trust tunnel). It significantly reduces the overhead of running a mesh by eliminating the need to over-provision memory and CPU while still ensuring security through mutual TLS, simple L4 authorization policies, and telemetry.
+
+The L7 proxies, known as waypoints, handle functions like traffic routing, authorization policy enforcement, and resilience. They run separately from application deployments and can scale independently based on demand.
+
+In contrast to the earlier injection of sidecars. Users can start with the secure L4 overlay, which offers features such as mTLS, authorization policy, and telemetry. Complex L7 handling such as retries, traffic splitting, load balancing, and observability collection can then be enabled on a case-by-case basis.
```

Comment:
Something seems to be wrong with this line ("In contrast to the earlier injection of sidecars."). Perhaps it should be part of the previous sentence?

Reply:
Hello Nir, The word `earlier` is confusing here. This sentence is not related to the previous L7 waypoint features. It compares the Ambient mode with the Istio Sidecar mode. I will rephrase it.